src/test/ui/span/borrowck-return-variable-on-stack-via-clone.stderr

```diff
@@ -0,0 +1,16 @@
+error: `x` does not live long enough
+  --> $DIR/borrowck-return-variable-on-stack-via-clone.rs:18:1
+   |
+17 |     (&x).clone() //~ ERROR `x` does not live long enough
+   |       - borrow occurs here
+18 | }
+   | ^ `x` dropped here while still borrowed
+   |
+note: borrowed value must be valid for the lifetime 'a as defined on the block at 16:30...
```

Comment:
Ideally we want to move away from this kind of message and move towards showing them where that lifetime ends rather than them having to look up what 16:30 means and trying to figure it out.

Can we do this patch without touching these errors, or are they part of what needs to be updated for this PR?